### PR TITLE
Slicer module

### PR DESCRIPTION
**Pull request by tashrifbillah**
Slicer can be built standalone or downloaded. The current `wm_apply_ORG_atlas_to_subject.sh` script is compatible for a built Slicer that does not require various paths adjustment during execution. But it is not compatible with path adjusted Slicer commands. The incompatibility arises with the values accepted by `-s` and `-m` arguments.

* The `-s` accepted by `wm_apply_ORG_atlas_to_subject.sh` may not be just `Slicer`. For a shared `Slicer`, it can be `"AdditionalPaths=/path/to/SlicerDMRI /path/to/etc Slicer"`.

* Again, for a downloaded (not built) Slicer the `-m` argument can be `"Slicer --launch /path/to/Slicer-4.10/cli-modules/FiberTractMeasurements"`.

To support all such values of `-s` and `-m`, [`$SlicerPath`](https://github.com/SlicerDMRI/whitematteranalysis/blob/master/bin/wm_apply_ORG_atlas_to_subject.sh#L298) and [`$FiberTractMeasurementsCLI`](https://github.com/SlicerDMRI/whitematteranalysis/blob/master/bin/wm_apply_ORG_atlas_to_subject.sh#L393) must be enclosed in double-quotes.

For the same multiple choices available to `-s`, it is safest to check the existence of the last element of `-s` i.e. `args.Slicer.split()[-1]`.

### CODE REVIEW

**Review by tashrifbillah — COMMENTED** [review]
Everything looks aligned in my machine:

image

File: bin/wm_apply_ORG_atlas_to_subject.sh [hunk 1]
```diff
@@ -128,14 +128,16 @@ else
 fi
 
 if [ $DiffMeasure == 1 ]; then
+    tmp=($FiberTractMeasurementsCLI)
 	if [ -z "$FiberTractMeasurementsCLI" ] ; then
```
[the remark this answers is not in the document]
Comment (tashrifbillah) [reply bin/wm_apply_ORG_atlas_to_subject.sh:132]: Hi @pieper, can we use your expert eyes about the undefined indentation problem?

It does look broken between lines [130 and 142](https://github.com/tashrifbillah/whitematteranalysis/blob/slicer-module/bin/wm_apply_ORG_atlas_to_subject.sh#L130) on GitHub.

[the remark this answers is not in the document]
Comment (zhangfanmark) [reply bin/wm_apply_ORG_atlas_to_subject.sh:132]: This is related to an inconsistency between the choices of using tab and space. I will fix this after testing you pull request. I am testing the code myself. Should be done soon.

**Comment by zhangfanmark** [comment]
@tashrifbillah we have merged your pull request and the indentation issue is fixed. thank you.

File: bin/wm_apply_ORG_atlas_to_subject.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (tashrifbillah) [reply bin/wm_apply_ORG_atlas_to_subject.sh:132]: Yup, that was my hunch but didn't quite get to checking it.

**Pull request merged**